src/components/imageGalleryMask.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+import ImageGalleryMask from './imageGalleryMask';
+
+export default {
+  title: 'Components/ImageGalleryMask',
```

Comment:
![image](https://github.com/user-attachments/assets/34caba80-3d70-4b6b-9772-2393e7dc018f)

Please added to Components/Images/ImageGalleryMask    Where Images is the category of the component